src/many/src/server/module/_1002_idstore.rs

```diff
@@ -14,7 +14,7 @@ pub use get::*;
 pub use store::*;
 pub use types::*;
 
-#[many_module(name = IdStoreModule, id = 1002, namespace = idstore, many_crate = crate)]
+#[many_module(name = IdStoreModule, id = 1002, namespace = idstore, many_crate = crate, drop_non_webauthn = [store])]
```

Comment:
Why not do `#[many(drop_non_webauthn)]` on the method itself, then in `Endpoint::new` check for it? Have a field in `Endpoint` and look if it's true. That would simplify the code no?

I'd prefer that approach but time being short, I'll let you create an issue to refactor and LGTM this (unless you think this approach is better).

Reply:
Good idea, thanks for the suggestion! I think your approach would be better. I will create an issue to refactor, thanks!